package.json

```diff
@@ -23,7 +23,7 @@
     "firebase",
     "realtime"
   ],
-  "main": "dist/angularfire.js",
+  "main": "index.js",
   "files": [
     "dist/**",
```

Comment:
I think we should also list `index.js` in the `files` list right here. It may not be required since npm may automatically include the file listed in `main`, but I'd prefer to see it listed in `files` as well.